02-eda.R

```diff
@@ -11,9 +11,9 @@ library(tibble)
 #1.  Find the top 10 starting integers that produce the longest sequences
 
 top10longest <- gen_collatz %>%
-                  arrange(desc(gen_collatz$length)) %>%
-                  select(gen_collatz$length(1:10) & gen_collatz$start) %>%
-                  print(gen_collatz$start)
+                arrange(desc(gen_collatz$length), .by_group = TRUE) %>%
+                select(gen_collatz$length(1:10) & gen_collatz$start) %>%
+                print(gen_collatz$start)
   
 # inspired from: 
 # https://statisticsglobe.com/select-top-n-highest-values-by-group-in-r
```

Comment:
Interesting! A good informative reference on how to find the highest values. The coding looks right, well done. Although, the simplest way to find the top 10 longest values is by using the ```slice_max``` function instead of manually selecting from the dataframe